dashboard/admin.py

```diff
@@ -79,5 +87,44 @@ def save_model(self, request, obj, form, change):
         obj.id = canvas_id_to_incremented_id(obj.canvas_id)
         return super(CourseAdmin, self).save_model(request, obj, form, change)
 
+class LogResource(resources.ModelResource):
+    class Meta:
+        model = Log
+
+# This is a local class for logs that adds in Export and disables some actions on the admin
+class MyLALogAdmin(ExportMixin, LogAdmin):
+    resource_class = LogResource
+    # Remove adding and editing for logs
+    @staticmethod
+    def has_add_permission(request):
+        return False
+    @staticmethod
+    def has_change_permission(request):
+        return False
+    @staticmethod
+    def has_delete_permission(request):
+        return False
+
+# This is local class for Cron that disables add
+class MyLACronJobLogAdmin(CronJobLogAdmin):
+    # Remove adding and editing for cron logs
+    @staticmethod
+    def has_add_permission(request):
+        return False
+    @staticmethod
+    def has_change_permission(request):
+        return False
+    @staticmethod
+    def has_delete_permission(request):
+        return False
```

Comment:
Same issue as above.
```suggestion
    @staticmethod
    def has_change_permission(request, obj=None):
        return False
    @staticmethod
    def has_delete_permission(request, obj=None):
        return False
```